lib/rpc/src/rpc_client.erl

```diff
@@ -448,24 +448,16 @@ make_call2(From, Size, Call, Timeout, S, Procedure, Timers0) ->
 	case Tm of
 	    infinity ->
 		Timers0;
-	    0 ->
-		Timers0;
 	    _ ->
 		Timer = erlang:send_after(Tm, self(), {timeout, S#state.xid}),
 		[Timer | Timers0]
 	end,
-    case Tm of
-	0 ->
-	    {reply, {error, timeout}, S#state{xid = S#state.xid+1}};
-	_ ->
-	    Pnew = #pending{from = From, start_time = now(),
-			    timers = Timers1,
-			    bytes_out = Size,
-			    xid = S#state.xid, packet = Call},
-	    S1 = S#state{xid = S#state.xid+1, pending = [Pnew|Ps],
-			 pendinglen = Plen + 1},
-	    {noreply, incr_call_stats(S1, Procedure)}
-    end.
+    Pnew = #pending{from = From, start_time = now(),
+		    timers = Timers1,
+                    bytes_out = Size, xid = S#state.xid, packet = Call},
+    S1 = S#state{xid = S#state.xid+1, pending = [Pnew|Ps],
+		 pendinglen = Plen + 1},
+    {noreply, incr_call_stats(S1, Procedure)}.
```

Comment:
what was the source version this pull request is based on?
It's definitely not the version from git